### PR TITLE
Added PremiumSoft.NavicatPremium 15.0.25

### PR DESCRIPTION
**Pull request by VAllens**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21023)

### CODE REVIEW

**Comment by VAllens** [comment]
I don't understand what the mistake is.

```console

7/15/2021 4:36:56 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: Started
============================================================================== 
Task         : Installation Validation
Description  : Invoke an Azure Function
Version      : 1.0.12
Author       : Microsoft Corporation
Help URL     : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/azure-function
============================================================================== 
2021-07-15T04:36:57.9519931Z POST ***?code=***&
Request body: {
  "operationId": "WinGetSvc-Validation-120-21023-20210715-1",
  "BuildId": "199692",
  "PlanUrl": "https://dev.azure.com/ms/",
  "HubName": "build",
  "ProjectId": "ed6a5dfa-6e7f-413b-842c-8305dd9e89e6",
  "PlanId": "20f930f8-a223-416d-8ac9-ed52ef9393a9",
  "JobId": "871b4eed-22de-5cb6-c1c8-82f190de633f",
  "TimelineId": "20f930f8-a223-416d-8ac9-ed52ef9393a9",
  "TaskInstanceId": "58bdf892-4e45-5ba7-4782-31912f95e167",
  "AuthToken": ***
}
				Response Code: OK
				Response: 

7/15/2021 4:36:59 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:37:33 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:38:25 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:38:58 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:39:49 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:40:43 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:41:14 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:41:55 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:42:25 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:42:58 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:43:29 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:43:59 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:44:36 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:45:22 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:46:19 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:47:18 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:48:10 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:48:41 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:49:33 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:50:28 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:50:58 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:51:55 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:52:47 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:53:38 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:54:30 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:55:26 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:56:01 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:56:52 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:57:43 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:58:19 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 4:59:15 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 5:00:11 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 5:01:08 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 5:01:39 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 5:02:09 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/15/2021 5:02:17 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: Completed
Result: Failed
```

**Comment by VAllens** [comment]
Hello. @wingetbot @OfficialEsco
I don't understand what the mistake is. 
can you please check it here?

File: manifests/p/PremiumSoft/NavicatPremium/15.0.25/PremiumSoft.NavicatPremium.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: PremiumSoft.NavicatPremium
+PackageVersion: 15.0.25
+PackageLocale: en-US
+Publisher: PremiumSoft CyberTech Ltd.
+PublisherUrl: https://www.navicat.com
+PublisherSupportUrl: https://www.navicat.com/support
+Author: Allen.Cai
+PackageName: Navicat Premium 15
```
Comment (OfficialEsco) [inline manifests/p/PremiumSoft/NavicatPremium/15.0.25/PremiumSoft.NavicatPremium.locale.en-US.yaml:11]:
```suggestion
PackageName: PremiumSoft Navicat Premium
```
image

File: manifests/p/PremiumSoft/NavicatPremium/15.0.25/PremiumSoft.NavicatPremium.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    InstallerType: inno
+    Scope: user
+    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_cs_x86.exe
+    InstallerSha256: 9660B59C6CA85092636FC85B6E4FAEE7585CCB3ED25AB247C9EC496E3C2769ED
+    ProductCode: "PremiumSoft.NavicatPremium"
+    InstallerSwitches:
+      Silent: /VERYSILENT
+      SilentWithProgress: /SILENT
+      Custom: /SUPPRESSMSGBOXES /CURRENTUSER
+    UpgradeBehavior: install
```
Comment (OfficialEsco) [inline manifests/p/PremiumSoft/NavicatPremium/15.0.25/PremiumSoft.NavicatPremium.installer.yaml:111]:
```suggestion
  - Architecture: x64
    InstallerLocale: en-US
    InstallerType: inno
    Scope: machine
    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_en_x64.exe
    InstallerSha256: 3B4BF7E3A077533CD103048AA8CF4ED8E8C3C9F60058EC708BB6390EE26F8CDC
    InstallerSwitches:
      Custom: /ALLUSERS
    UpgradeBehavior: install
  - Architecture: x64
    InstallerLocale: en-US
    InstallerType: inno
    Scope: user
    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_en_x64.exe
    InstallerSha256: 3B4BF7E3A077533CD103048AA8CF4ED8E8C3C9F60058EC708BB6390EE26F8CDC
    InstallerSwitches:
      Custom: /CURRENTUSER
    UpgradeBehavior: install
  - Architecture: x86
    InstallerLocale: en-US
    InstallerType: inno
    Scope: machine
    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_en_x86.exe
    InstallerSha256: A44C61559022D7545354C68530A103C1BAB6A84F75BDFB869743CA6F2833DE13
    InstallerSwitches:
      Custom: /ALLUSERS
    UpgradeBehavior: install
  - Architecture: x86
    InstallerLocale: en-US
    InstallerType: inno
    Scope: user
    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_en_x86.exe
    InstallerSha256: A44C61559022D7545354C68530A103C1BAB6A84F75BDFB869743CA6F2833DE13
    InstallerSwitches:
      Custom: /CURRENTUSER
    UpgradeBehavior: install
  - Architecture: x64
    InstallerLocale: zh-CN
    InstallerType: inno
    Scope: machine
    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_cs_x64.exe
    InstallerSha256: 6B185DDC1B65841A83372EF4796F9A32836164EC923F84C959B0B9DCE662634E
    InstallerSwitches:
      Custom: /ALLUSERS
    UpgradeBehavior: install
  - Architecture: x64
    InstallerLocale: zh-CN
    InstallerType: inno
    Scope: user
    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_cs_x64.exe
    InstallerSha256: 6B185DDC1B65841A83372EF4796F9A32836164EC923F84C959B0B9DCE662634E
    InstallerSwitches:
      Custom: /CURRENTUSER
    UpgradeBehavior: install
  - Architecture: x86
    InstallerLocale: zh-CN
    InstallerType: inno
    Scope: machine
    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_cs_x86.exe
    InstallerSha256: 9660B59C6CA85092636FC85B6E4FAEE7585CCB3ED25AB247C9EC496E3C2769ED
    InstallerSwitches:
      Custom: /ALLUSERS
    UpgradeBehavior: install
  - Architecture: x86
    InstallerLocale: zh-CN
    InstallerType: inno
    Scope: user
    InstallerUrl: http://download3.navicat.com/download/navicat150_premium_cs_x86.exe
    InstallerSha256: 9660B59C6CA85092636FC85B6E4FAEE7585CCB3ED25AB247C9EC496E3C2769ED
    InstallerSwitches:
      Custom: /CURRENTUSER
    UpgradeBehavior: install
```
* Wrong ProductCode
* Cleaned up unnecessary switches

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Its probably the bot being stupid, higher chance with more installers...
```
Found Navicat Premium 15 [PremiumSoft.NavicatPremium]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading http://download3.navicat.com/download/navicat150_premium_en_x64.exe
  ██████████████████████████████  58.5 MB / 58.5 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

File: manifests/p/PremiumSoft/NavicatPremium/15.0.25/PremiumSoft.NavicatPremium.installer.yaml (hunk 2 above)
Comment (VAllens) [reply manifests/p/PremiumSoft/NavicatPremium/15.0.25/PremiumSoft.NavicatPremium.installer.yaml]: Okay...
Thanks

**Comment by VAllens** [comment]
Good. :)  
Thanks for @OfficialEsco 👍

**Comment by VAllens** [comment]
Hi, @OfficialEsco 
I don't know what happened, it seems to have no errors and warnings.
Can you check it for me again?
Thanks you. 

```console
Build #WinGetSvc-Validation-120-21023-20210719-2 failed
0 errors / 0 warnings

Installer Validation failed
0 errors / 0 warnings
```

**Comment by denelon** [comment]
@VAllens you have to dig just a bit deeper. If you click on the "0 errors / 0 warnings" link in the build, you will get to the detail.

https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=201428&view=logs&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=58bdf892-4e45-5ba7-4782-31912f95e167

**Comment by VAllens** [comment]
Hello, @denelon, thank you for your help. :)

I looked at the detailed log, but the log seemed to miss the key information...

```console

7/19/2021 1:37:40 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: Started
============================================================================== 
Task         : Installation Validation
Description  : Invoke an Azure Function
Version      : 1.0.12
Author       : Microsoft Corporation
Help URL     : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/azure-function
============================================================================== 
2021-07-19T01:37:41.7402827Z POST ***?code=***&
Request body: {
  "operationId": "WinGetSvc-Validation-120-21023-20210719-2",
  "BuildId": "201428",
  "PlanUrl": "https://dev.azure.com/ms/",
  "HubName": "build",
  "ProjectId": "ed6a5dfa-6e7f-413b-842c-8305dd9e89e6",
  "PlanId": "d4f79020-f22d-46b8-a534-4ae7f52b8725",
  "JobId": "871b4eed-22de-5cb6-c1c8-82f190de633f",
  "TimelineId": "d4f79020-f22d-46b8-a534-4ae7f52b8725",
  "TaskInstanceId": "58bdf892-4e45-5ba7-4782-31912f95e167",
  "AuthToken": ***
}
				Response Code: OK
				Response: 

7/19/2021 1:37:43 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:38:40 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:39:14 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:40:05 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:41:02 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:41:56 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:42:51 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:43:39 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:44:34 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:45:26 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:46:13 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:47:07 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:47:50 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:48:40 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:49:30 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:50:19 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:50:52 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:51:42 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:52:38 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:53:13 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:53:57 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:54:54 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:55:43 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:56:24 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:57:21 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:58:08 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:59:03 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 1:59:56 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 2:00:48 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: InProgress

7/19/2021 2:00:56 AM

****************************************************************************************
Installation Verification
****************************************************************************************
Status: Completed
Result: Failed
```

**Comment by denelon** [comment]
@VAllens I was hoping it would pass the second time through. I'll have to get someone to dig into the details and see what is happening.

**Comment by VAllens** [comment]
Thanks for @denelon and @ChuckFerring. 👍

**Review by OfficialEsco — APPROVED** [review]

**Comment by ItzLevvie** [comment]
cc @denelon @palenshus: "Azure-Pipeline-Passed" label needs to be added to this PR.

**Comment by denelon** [comment]
@ItzLevvie I think we have a bug in our logic. I see @OfficialEsco approved this PR earlier. I'm going to approve to see if this will get the bot to add the "Moderator-Approved" label and merge.

**Review by denelon — APPROVED** [review]

**Comment by VAllens** [comment]
Very gooooooooooooooooooooooooooooood!!! 💯

**Pull request merged**